src/components/ha-wa-dialog.ts

```diff
@@ -52,8 +54,8 @@ export type DialogWidth = "small" | "medium" | "large" | "full";
  * @attr {boolean} open - Controls the dialog open state.
  * @attr {("small"|"medium"|"large"|"full")} width - Preferred dialog width preset. Defaults to "medium".
  * @attr {boolean} prevent-scrim-close - Prevents closing the dialog by clicking the scrim/overlay. Defaults to false.
- * @attr {string} header-title - Header title text when no custom title slot is provided.
- * @attr {string} header-subtitle - Header subtitle text when no custom subtitle slot is provided.
+ * @attr {string|boolean} header-title - Header title text, or true to enable the headerTitle slot.
+ * @attr {string|boolean} header-subtitle - Header subtitle text, or true to enable the headerSubtitle slot.
```

Comment:
This is weird. Can't we have it as `?string` and add the slot if it's undefined?